package.json

```diff
@@ -21,13 +21,13 @@
   "dependencies": {
     "@alaskaairux/icons": "^4.25.0",
     "@aurodesignsystem/auro-dropdown": "^2.9.6",
-    "@aurodesignsystem/auro-input": "^2.13.1",
+    "@aurodesignsystem/auro-input": "^2.14.0",
```

Comment:
Revert back to v2.13.1. A bug was introduced with the last release that is causing an accessibility violation

https://github.com/AlaskaAirlines/auro-datepicker/actions/runs/4962714746/jobs/8881107808#step:5:121

```
❌ auro-datepicker > auro-datepicker is accessible
      Error: Accessibility Violations
      ---
      Rule: label-title-only
      Impact: serious
      Form elements should have a visible label (https://dequeuniversity.com/rules/axe/4.7/label-title-only?application=axeAPI)
      
      Issue target: auro-datepicker,auro-input,input
      Context: <input part="input" class=" inputIcon inputElement " id="" type="month-day-year" inputmode="numeric" aria-describedby="0ecv6d" aria-invalid="true" placeholder="MM/DD/YYYY" lang="">
      Fix all of the following:
        Only title used to generate label for form element
      ---
        at processResults (node_modules/chai-a11y-axe/src/accessible.js:87:15)
        at node_modules/chai-a11y-axe/src/accessible.js:[123](https://github.com/AlaskaAirlines/auro-datepicker/actions/runs/4962714746/jobs/8881107808#step:5:124):7
        at async o.<anonymous> (test/auro-datepicker.test.js:10:5)
```

Reply:
What is weird is that this issues only comes up when you move to the latest version of auro-input. Nothing functional changed with this version. 

If you could shed any light on why this may be happening inside auro-datepicker it would be appreciated.